samples/bluetooth/subrating/README.rst

```diff
@@ -0,0 +1,225 @@
+.. _ble_subrating:
+
+Bluetooth: Connection Subrating
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Bluetooth® Connection Subrating sample showcases the effect of the LE Connection Subrating feature on the duty cycle of a connection.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+You can use any two of the development kits mentioned above and mix different development kits.
+
+Additionally, the sample requires a connection to a computer with a serial terminal for each of the development kits.
+
+Overview
+********
+
+Subrating provides a mechanism for indicating that only a specific subset of connection events are to be actively used by the connected devices, with the radio not being used in other connection events.
+Connection events are actively used when the peripheral listens to the event or sends data in the event.
```

Comment:
```suggestion
Connection events are actively used when the peripheral listens for an event or sends data contained in the event.
```
Just guessing here

Reply:
In this case the peripheral will listen to specific events as determined by the subrate configuration and send its data in them, I think the more specific wording makes it a bit clearer. I replaced `to` with `for`, does the rest seem ok to you?